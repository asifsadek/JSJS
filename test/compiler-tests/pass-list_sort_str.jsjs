val slist1 = ["foo", "bar", "baz"];

// compare functions
val lt = /\(x, y) => x < y;
val gt = /\(x,y) => x > y;

List.iter(print, List.sort(lt, slist1));
List.iter(print, List.sort(/\(x, y) => x > y, slist1));

