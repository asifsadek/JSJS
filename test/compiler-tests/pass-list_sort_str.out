bar
baz
foo
foo
baz
bar
